### PR TITLE
182 conformance and normative terms

### LINKED ISSUE #182: Conformance and Normative terms
1. Add sentence specifying the use of normative language, per RFC2119 
2. Check all appearances of the keywords to determine if they should be capitalized or not. 

e.g. DCAT has the following statement: 

> The key words MAY, MUST, MUST NOT, and SHOULD in this document are to be interpreted as described in [BCP 14](https://tools.ietf.org/html/bcp14) [[RFC2119](https://www.w3.org/TR/vocab-dcat-2/#bib-rfc2119)] [[RFC8174](https://www.w3.org/TR/vocab-dcat-2/#bib-rfc8174)] when, and only when, they appear in all capitals, as shown here.

### PR DESCRIPTION
**Pull request by dr-shorthair**
Closes #182

### CODE REVIEW

**Comment by dr-shorthair** [comment]
@sgrellet @hylkevds @ldesousa 
Can you review this please? It is a minor addition that will get rid of one of the respec warnings. 

Would also be interested in your additions to the actual content of the Conformance statement - see https://github.com/w3c/sdw-sosa-ssn/pull/198/files#diff-380de27307896ad97825cbc8b0794c863bb203aa37c80d2351b27b9982e3b204

**Comment by ldesousa** [comment]
> @sgrellet @hylkevds @ldesousa Can you review this please? It is a minor addition that will get rid of one of the respec warnings.
> 
> Would also be interested in your additions to the actual content of the Conformance statement - see https://github.com/w3c/sdw-sosa-ssn/pull/198/files#diff-380de27307896ad97825cbc8b0794c863bb203aa37c80d2351b27b9982e3b204

My agenda is too heavy the next two days. I will be having a look on the 6th of March. That's about 32 hours before the next meeting.

**Review by hylkevds — APPROVED** [review]
LGTM.

File: ssn/index.html [hunk 1]
```diff
@@ -155,7 +167,7 @@ <h3 id="sosa-module">SOSA</h3>
       properties, and provides textual definitions and other annotations.
       Some semantic enrichment is provided using Schema.org [[schema-org]] properties, but there is no formal
       axiomatization.
-      SOSA may be used <i>as-is</i> by an audience who wish to use terminology consistent with the SSN Ontology, thus
+      SOSA MAY be used <i>as-is</i> by an audience who wish to use terminology consistent with the SSN Ontology, thus
```
Comment (ldesousa) [inline ssn/index.html:170]: This makes "MAY" look like an acronym. I suggest using bold, italics, underlined or a combination of those to highlight these terms.

File: ssn/index.html [hunk 2]
```diff
@@ -606,7 +620,7 @@ <h3>Quantity Values and Unit of Measures</h3>
         </li>
         <li>With OM 2, a quantitative result would be a <a
             href="http://www.wurvoc.org/vocabularies/"><code>om:Measure</code> or <code>om:Point</code></a></li>
-        <li>With CDT, the value of <a href="#SOSAhasSimpleResult"><code>sosa:hasSimpleResult</code></a> may be
+        <li>With CDT, the value of <a href="#SOSAhasSimpleResult"><code>sosa:hasSimpleResult</code></a> can be
```
Comment (ldesousa) [inline ssn/index.html:623]: Why is "MAY" highlighted, and "can" not?

**Review by ldesousa — CHANGES_REQUESTED** [review]
Small things for further thought. Fine for me if you merge outright.

In parallel I pushed some additions to the conformance articles in Conformance.html, please cross-review those.

File: ssn/index.html (hunk 2 above)
Comment (dr-shorthair) [reply ssn/index.html:623]: MAY is a normative term. 
'can' is not (according to the IETF standard). 

But this whole section is to be replaced by work that I'm doing on the Patterns section in another branch.

File: ssn/index.html (hunk 1 above)
Comment (dr-shorthair) [reply ssn/index.html:170]: The Conformance rules require that the normative terms be cast into UPPER CASE where they are being used normatively.

**Comment by dr-shorthair** [comment]
Thanks @ldesousa  - your mods are good. 
Can you release this now?

**Review by ldesousa — APPROVED** [review]

**Pull request merged**